src/witan/cic/tuairisc/date_utils.clj

```diff
@@ -13,3 +16,18 @@
 
 (defn year-week [date]
   (tick/with date :day-of-week 1))  ;; Move everything to the Monday of the week
+
+(defn overlaps-window?
+  "
+                 ws        we
+   false: ps pe
+    true: ps       pe
+    true:          ps  pe
+    true:              ps    pe
+   false:                    ps  pe
+    true:    ps              pe  "
```

Comment:
What does this mean?

Reply:
ws/we window start window end
ps/pe period start period end

It is sort of graphically showing the logic of when it is true and false as I was getting my head in a tangle.

Reply:
okay I can see what it is now with that explanation - does it need that adding to it to explain why it's working like this or even a dsdr even if it is a super simple decision?